community/values/src/main/java/org/neo4j/values/storable/NoValue.java

```diff
@@ -28,7 +28,7 @@
  * The NULL object of the Value world. Is implemented as a singleton, to allow direct reference equality checks (==),
  * and avoid unnecessary object creation.
  */
-final class NoValue extends Value
+public final class NoValue extends Value
```

Comment:
Why do we make this public? No one should rely on this class but only on `Values.NO_VALUE`.

Reply:
Hmm it looks like I used the eq-method in one place and an instanceof-check in another, but it was just being silly. I will replace with == and make this package private again.